spec/unit/run_list/versioned_recipe_list_spec.rb

```diff
@@ -187,4 +187,9 @@
     end
   end
 
+  context "with duplicated names", :chef_gte_13_only do
+    it "should fail in Chef 13" do
+      expect(list.with_duplicate_names).to raise_error
```

Comment:
Also, if we're expecting `NoMethodError`, it's best to specify that explicitly, or you could change it to `expect(thing).to_not respond_to(:method)`